### ISSUE TITLE
mkstr and symb

### ISSUE THREAD

**Issue by EuAndreh** [tarballs-are-good/quickutil#39]
```lisp
(defun mkstr (&rest args)
  "Receives any number of objects (string, symbol, keyword, char, number),
   extracts all 'string desigators' and concatenates them all into one string.
   Extracted from On Lisp, chapter 4."
  (with-output-to-string (s)
    (dolist (a args) (princ a s))))

(defun symb (&rest args)
  "Receives any number of objects, concatenates all into one string with #'mkstr
   and converts them to symbol.
   Extracted from On Lisp, chapter 4."
  (values (intern (apply #'mkstr args))))
```

Provides: symb, mkstr
Requires: symb depends on mkstr, nothing else
Author: Paul Graham.
License: He published it on the book, and then opened the book on his website, so I believe it must be BSD-like.

It is better than Alexandria's symbolicate: it accepts as arguments numbers, lists, vectors, hash tables, unreadable objects or anything else.

**Comment by marzipankaiser** [comment]
I like those, too. I remember that there were quite some more utilities in his book, maybe someone could ask him about the license, most are probably worth adding...

**Comment by tarballs-are-good** [comment]
Sorry about the late additions. Thanks, and happy holidays!

If there are more from Paul Graham's book that are desired, please make a new issue. :)
